src/NuGet.Clients/NuGet.SolutionRestoreManager/VsSolutionRestoreService.cs

```diff
@@ -26,22 +29,30 @@ namespace NuGet.SolutionRestoreManager
     /// </summary>
     [PartCreationPolicy(CreationPolicy.Shared)]
     [Export(typeof(IVsSolutionRestoreService))]
-    internal sealed class VsSolutionRestoreService : IVsSolutionRestoreService
+    public sealed class VsSolutionRestoreService : IVsSolutionRestoreService
     {
         private const string IncludeAssets = "IncludeAssets";
         private const string ExcludeAssets = "ExcludeAssets";
         private const string PrivateAssets = "PrivateAssets";
+        private const string TargetFrameworks = "TargetFrameworks";
```

Comment:
Nowhere, actual property value is mapped to the `IVsProjectRestoreInfo.OriginalTargetFrameworks`. This property is treated differently as it shouldn't be evaluated per each tfm.